notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.        # Since there are special rules for the GISS model it has to be done like this. Think of ways to make it simpler
</pre><p>If there is no way to simplify this, consider adding a few lines of comment to explain why this is required. The code all looks fine, but would save the viewer some time to know why this has to be done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done</p>

Reply:
closing conversation, resolved